tools/linter_lib/custom_check.py

```diff
@@ -484,6 +484,11 @@ def build_custom_checkers(by_lang):
          ]),
          'description': 'Comment-style function type annotation. Use Python3 style annotations instead.',
          },
+        {'pattern': 'null=True',
```

Comment:
I might write this as `= models[.].*null=True` to make it a bit less spammy

Reply:
I've also rewritten the original check to avoid the `exclude_pattern`, which appears to still work, but I couldn't get that working for the combination in the second form.